<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eDKit alpha (c) 2014 by bailli</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDKEdit" name="lvlEdit" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>512</width>
      <height>448</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbLevel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>326</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="lvlInfo">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>464</y>
      <width>511</width>
      <height>201</height>
     </rect>
    </property>
    <property name="plainText">
     <string>das ist ein text
zeile 2
3
4
5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWrite">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>390</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Write level data</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tabTiles">
     <attribute name="title">
      <string>&amp;Tiles</string>
     </attribute>
     <widget class="QTileSelector" name="tileSelect" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>8</y>
        <width>256</width>
        <height>256</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSprites">
     <attribute name="title">
      <string>&amp;Sprites</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Level:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_ROM"/>
    <addaction name="actionSave_ROM"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen_ROM">
   <property name="text">
    <string>&amp;Open ROM</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionSave_ROM">
   <property name="text">
    <string>&amp;Save ROM</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDKEdit</class>
   <extends>QWidget</extends>
   <header>QDKEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QTileSelector</class>
   <extends>QWidget</extends>
   <header>QTileSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
